spec/jobs/searchgov_url_fetcher_job.rb

```diff
@@ -1,7 +1,7 @@
 require 'spec_helper'
 
 describe SearchgovUrlFetcherJob do
-  subject(:perform) { SearchgovUrlFetcherJob.perform_now(args[:searchgov_url]) }
+  subject(:perform) { SearchgovUrlFetcherJob.perform_now(searchgov_url) }
   let!(:searchgov_domain) { SearchgovDomain.create(domain: 'agency.gov', status: '200') }
   let(:searchgov_url) { SearchgovUrl.create(url: 'https://agency.gov/') }
   let(:args) do
```

Comment:
Per my earlier comment, you can remove the `args` variable, which you are no longer using.

Reply:
I believe the `args` variable is necessary for the shared example, `it_behaves_like 'a searchgov job'`, to work correctly?

Reply:
Ah, you are correct. I was thinking you could just pass the `searchgov_url` record as the argument, rather than a hash, because you are not using keyword arguments.

However...let's change that. I am not always consistent about using keyword args, but I do like how explicit they are about complaining when you are missing an argument. Using keyword args would also be consistent with the other job arguments:
```
$ gg 'def perform' app/jobs/
app/jobs/searchgov_domain_indexer_job.rb:  def perform(searchgov_domain:, delay:)
app/jobs/searchgov_domain_preparer_job.rb:  def perform(searchgov_domain:)
app/jobs/sitemap_indexer_job.rb:  def perform(searchgov_domain:)
```
So, instead of:
`def perform(searchgov_url)`

You'd use:
`def perform(searchgov_url:)`

And the way to initiate that job would be:
`SearchgovUrlFetcherJob.perform_now(searchgov_url: searchgov_url)`

It's a little more wordy than just `SearchgovUrlFetcherJob.perform_now(searchgov_url)`, but it gives us more flexibility in case we need to add  more args down the line. Sorry for making this specification late in the game; I was thinking I didn't mind the inconsistency with the other jobs, but I do think we should make this change.

Reply:
Gotcha.  So I now have it that `perform` uses a keyword argument.  Since the `args` variable is still necessary, would it be cleaner to write `SearchgovUrlFetcherJob.perform_now(args)` in `subject` declaration in the spec?

Reply:
yep